src/main/java/org/prebid/server/floors/BasicPriceFloorResolver.java

```diff
@@ -254,6 +260,8 @@ private PrebidConfigParameter createParameter(PriceFloorField field,
             case adUnitCode -> adUnitCodeFromImp(imp);
             case country -> countryFromRequest(bidRequest);
             case deviceType -> resolveDeviceTypeFromRequest(bidRequest);
+            //todo: do we need to check whether the imp contains the bidder?
+            case bidder -> SimpleDirectParameter.of(bidder);
```

Comment:
I guess we don't need to check it, because the one that called price floor function gets this bidder from imp in some way of abstraction.

Reply:
ok